.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: node_js
 node_js:
-- '8'
+  - '8'
 script:
-- yarn jest
-- yarn lint
+  - yarn jest
+  - yarn lint
+  - yarn type-check
```

Comment:
This is the only change; rest is prettier

Reply:
👍